### PR TITLE
Changed import format.

### PR DESCRIPTION
**Pull request by normanjaeckel**
As discussed yesterday this PR changes the format for initial-data and import. Every collection is now a dict instead of a list. The PR for the main repo will follow soon.

### CODE REVIEW

File: openslides_backend/models/checker.py [hunk 1]
```diff
@@ -270,7 +269,11 @@ def run_check(self) -> None:
         self.check_json()
         self.check_collections()
         for collection, models in self.data.items():
-            for model in models:
+            for id_, model in models.items():
+                if model["id"] != int(id_):
+                    self.errors.append(
```
[the remark this answers is not in the document]
Comment (normanjaeckel) [reply openslides_backend/models/checker.py:274]: Done.

File: openslides_backend/action/actions/meeting/import_.py [hunk 2]
```diff
@@ -131,17 +130,24 @@ def check_usernames_and_generate_new_ones(self, json_data: Dict[str, Any]) -> No
...
 
         # set enable_anonymous
-        json_data["meeting"][0]["enable_anonymous"] = False
+        self.get_meeting_from_json(json_data)["enable_anonymous"] = False
 
         # set imported_at
-        json_data["meeting"][0]["imported_at"] = round(time.time())
+        self.get_meeting_from_json(json_data)["imported_at"] = round(time.time())
+
+    def get_meeting_from_json(self, json_data: Any) -> Any:
```
[the remark this answers is not in the document]
Comment (normanjaeckel) [reply openslides_backend/action/actions/meeting/import_.py:145]: I would like to leave it as is for the moment. The code can be enhanced in several points and I think one can better understand the current state with the more explicit (but repetitive) way.

**Comment by normanjaeckel** [comment]
Ready to merge.

**Review by jsangmeister — APPROVED** [review]
Looks good. Please only merge after the main repo is updated.

**Pull request merged**